Nimda/Core/Events/WelcomeMessage.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Nimda\Core\Events;
+
+use CharlotteDunois\Yasmin\Models\GuildMember;
+use Nimda\Core\UserEvent;
+use \CharlotteDunois\Yasmin\Models\MessageEmbed;
+
+class WelcomeMessage extends UserEvent
+{
+    public function userEventTrigger(GuildMember $member, GuildMember $memberOld = null)
+    {
+        if ($this->config['enabled'] === false || $this->config['channel'] === '') {
+            return;
+        }
+
+        $channel = $member->guild->channels->get($this->config['channel']);
+
+        if ($this->config['mention'] === true) {
+            $channel->send("Welcome to {$member->guild->name}, {$member}");
+        } elseif ($this->config['embed'] === true)
+        {
+            $embed = new MessageEmbed();
+            $embed
+                ->setTitle('Member Joined!')
```

Comment:
[PSR-2 method-and-function-calls](https://www.php-fig.org/psr/psr-2/#46-method-and-function-calls)